### ISSUE TITLE
Mock dynamic content

### ISSUE THREAD

**Issue by ms88privat** [NimaSoroush/differencify#56]
We need something similar to [freezeImage](https://github.com/NimaSoroush/differencify/blob/2034a2c3e2e89776f83f09e0cd56d740b6d27343/src/freezeImage.js) for general dynamic content. e.g `dates, timeAgo, ...`

**Comment by NimaSoroush** [comment]
Yeah, those things could be added as feature function. Feel free to raise a PR for it
